.github/workflows/ruskwallet_ci.yml

```diff
@@ -1,110 +0,0 @@
-name: rusk-wallet CI
-
-on:
-  pull_request:
-    types:
-      - opened
-      - synchronize
-      - reopened
-      - ready_for_review
-
-jobs:
-  # JOB to run change detection
-  changes:
-    runs-on: ubuntu-latest
-    # Required permissions
-    permissions:
-      pull-requests: read
-    # Set job outputs to values from filter step
-    outputs:
-      run-ci: ${{ steps.filter.outputs.run-ci }}
-    steps:
-      # For pull requests it's not necessary to checkout the code
-      - uses: dorny/paths-filter@v3
-        id: filter
-        with:
-          filters: |
-            run-ci:
-              - 'wallet-core/**'
-              - 'execution-core/**'
-              - 'rusk-wallet/**'
-              - '.github/workflows/ruskwallet_ci.yml'
-  fmt:
-    needs: changes
-    if: needs.changes.outputs.run-ci == 'true'
-    name: Rustfmt
-    runs-on: core
-    steps:
-      - uses: actions/checkout@v4
-      - uses: dsherret/rust-toolchain-file@v1
-      - run: cargo fmt --all -- --check
-
-  analyze:
-    needs: changes
-    if: needs.changes.outputs.run-ci == 'true'
-    name: Dusk Analyzer
-    runs-on: core
-    steps:
-      - uses: actions/checkout@v4
-      - uses: dsherret/rust-toolchain-file@v1
-      - run: cargo install --git https://github.com/dusk-network/cargo-dusk-analyzer
-      - run: cargo dusk-analyzer
-
-  test_nightly-linux:
-    needs: changes
-    if: needs.changes.outputs.run-ci == 'true'
-    name: "[Linux] Nightly tests"
-    runs-on: core
-
-    steps:
-      - uses: actions/checkout@v4
-      - uses: dsherret/rust-toolchain-file@v1
-      - run: cargo test --release
-        working-directory: ./rusk-wallet
-      - run: cargo clippy --all-features --release -- -D warnings
-        working-directory: ./rusk-wallet
-
-  test_nightly-macintel:
-    needs: changes
-    if: needs.changes.outputs.run-ci == 'true'
-    name: "[Mac Intel] Nightly tests"
-    runs-on: macos-latest
-
-    steps:
-      - uses: actions/checkout@v4
-      - uses: dsherret/rust-toolchain-file@v1
-
-      - name: Add arm target for Apple Silicon build
-        run: rustup target add aarch64-apple-darwin
-
-      - run: cargo test --release
-        working-directory: ./rusk-wallet
-
-  test_nightly-macm1:
```

Comment:
With this change we're no longer testing if  `rusk-wallet` works against Mac and Windows, since we're only running on core through Rusk-ci. Not sure if that's the intent?

Reply:
You're right, good catch